gulpfile.js

```diff
@@ -0,0 +1,48 @@
+/* global require */
+'use strict';
+
+var gulp = require('gulp');
+var jsdoc = require('gulp-jsdoc');
+var clean = require('gulp-rimraf');
+var babel = require('gulp-babel');
+var fs = require('fs');
+var path = require('path');
+
+var JSDOC_JSON = 'jsdoc.json';
+var JSDOC_TASK_PREFIX = 'jsdoc:';
+
+var jsdocTasks = [];
+var files = fs.readdirSync('apps');
+files.forEach(function(filePath, i) {
```

Comment:
line 16 to 40  wii generate ` jsdocTasks` list (jsdoc:bluetooth, jsdoc:system....) , which is pre-executed in `gulp docs` task. So it's not proper to wrapped into the `docs` task.